.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "azureResourceManagerTools.checkForLatestSchema": false
+}
```

Comment:
can you explain the reason of doing this? Thanks!

Reply:
To avoid annoying warnings about old schema when open ARM templates from the example directory.